CalibTracker/SiStripHitResolution/interface/HitResol.h

```diff
@@ -0,0 +1,179 @@
+#include "CalibFormats/SiStripObjects/interface/SiStripQuality.h"
```

Comment:
This HitResol.h file misses and header guard

Reply:
https://github.com/cms-sw/cmssw/pull/40754/commits/01bbe6e597150b216293e2dc33cdada65a192d3d.